iron-autogrow-textarea.html

```diff
@@ -125,6 +134,50 @@
       },
 
       /**
+       * Bound to the textarea's `autocomplete` attribute.
+       */
+      autocomplete: {
```

Comment:
Yeah, this commit adds something to bind the property to, since the `iron-autogrow-textarea` is in the `paper-textarea`'s local DOM.